website/static/js/pages/project-settings-page.js

```diff
@@ -54,8 +54,73 @@ if ($('#wgrid').length) {
         });
     });
 }
+
+
+var institutionsViewModel = function() {
```

Comment:
In general, try to keep the `*-page.js` modules relatively "thin"--they should only contain initialization code. ViewModels and other reusable code should be moved to their own modules.

> - Reuseable modules go in website/static/js/. Name modules in lowerCamelCase.
> - Initialization code for a page goes in a module within website/static/js/pages/. Name page modules with lower-dashed-case.

https://cosdev.readthedocs.org/en/latest/osf/js_modules.html#osf-js-modules